dockerfiles/centos-8

```diff
@@ -17,16 +17,16 @@
 # NOTE: These are intended to be built using the arguments as
 # described in ../build.sh. See that script for more details.
 
-FROM balenalib/aarch64-debian:stretch-build
+FROM centos:8
 
 # Choose whether to install SpiderMonkey 1.8.5, default yes
 ARG js=js
 # Choose whether to install Erlang, default yes
 ARG erlang=erlang
 # Select version of Node, Erlang and Elixir to install
-ARG erlangversion=19.3.6
-ARG elixirversion=v1.6.6
-ARG nodeversion=8
+ARG erlangversion=20.3.8.22-1
```

Comment:
just noting that there isn't an rpm for Erlang `20.3.8.22-1` - we'd need to bump this to `20.3.8.23` to avoid building from source.

Reply:
Right, cut and paste error. Will address

Reply:
FYI Erlang Solutions doesn't have a package for 20.x CentOS 8 anyway, so we're building from source here anyway.

Reply:
I'll fix it for 6 and 7.

Reply:
Fixed for 8 to 23.